packages/ui/src/lib/sidebar/SidebarSection.svelte

```diff
@@ -0,0 +1,8 @@
+<script>
+    import SidebarSectionTitle from "./SidebarSectionTitle.svelte";
+    import SidebarDivider from "./SidebarDivider.svelte";
+</script>
+
+<slot />
+
+<SidebarDivider />
```

Comment:
Do we really need this component? We can just use `<SidebarDivider />` directly to separate content.